spinnaker/api/models.go

```diff
@@ -0,0 +1,33 @@
+package api
+
+// PipelineTemplateV2 defines the schema of the pipeline template JSON.
+type PipelineTemplateV2 struct {
+	ID        string                       `json:"id,omitempty"`
+	Metadata  PipelineTemplateV2Metadata   `json:"metadata"`
+	Pipeline  interface{}                  `json:"pipeline"`
```

Comment:
What is the reason to not have the schema for Pipeline, too?

Reply:
I opted to not do that now as it contains lots of fields and subtypes. I'd like to tackle this when refactoring the pipeline resource logic.

For now I just wanted to have the schema for the pipeline template specific top level fields in order to validate them on plan.

Reply:
I think it is fine to postpone it a bit as you suggested.